collections/dominode-resources/checklists/legacy-vector.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "DomiNode - legacy vector validation checklist",
+  "description": "A checklist for validating legacy vector datasets",
+  "dataset_type": "vector",
+  "validation_artifact_type": "dataset",
+  "checks": [
+    {
+      "name": "legacy",
+      "description": "Check for legacy vector datasets.",
+      "guide": "This check will always defaults to True, it has been added for the validation of DomiNode legacy vector datasets",
+     "automation": {
+        "algorithm_id": "script:legacy",
```

Comment:
You need to update this to reflect the new name, which seems to be `script:noopvalidator` (please test it out)